src/styles/utils/variables.scss

```
@@ -0,0 +1,11 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$blue-color: #00ACDC;
+$white-color: #fff;
+$darkblue-color: #34568b;
+$light-grey-color: #f3f3f3;
+$secondary-line-height: 18px;
+$fixed-width: 166px;
```

No need to hardcode fixed width
```suggestion
$fixed-width: 100%;
```